.irbrc

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Is this still needed?

Reply:
I think they are https://github.com/standardrb/standard/pull/181#issuecomment-635722698